src/store.js

```diff
@@ -0,0 +1,29 @@
+import {createStore, applyMiddleware, compose} from 'redux';
+import rootReducer from './reducers';
+import ReduxThunk from 'redux-thunk/';
+import {persistStore, persistReducer} from 'redux-persist';
+import storage from 'redux-persist/lib/storage';
+import autoMergeLevel1 from 'redux-persist/lib/stateReconciler/autoMergeLevel1';
+
+const persistConfig = {
+    key: 'root',
+    storage: storage,
+    // autoMergeLevel1 is the default anyway, but I want to be explicit
+    stateReconciler: autoMergeLevel1,
+    whitelist: ['auth', 'map']
+};
+
+const persistedReducer = persistReducer(persistConfig, rootReducer);
```

Comment:
The `persistedReducer` is now a persistent version of our global state reducer. It's going to take as arguments the `persistConfig` that we configured above, and then our `rootReducer` which it can reach into to grab the appropriate whitelisted pieces of global state to persist.